### DISCUSSION TITLE
Prisma error when run Generate

### DISCUSSION THREAD

**Discussion by mcjambi** [prisma/prisma#22701] [Q&A]
### Bug description

after run command: "npx prisma generate" i meet error:
```
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Error: request to https://binaries.prisma.sh/all_commits/78caf6feeaed953168c64e15a249c3e9a033ebe2/windows/query_engine.dll.node.sha256 failed, reason: write EPROTO C0290000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:355:
```
Please fix it!

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @mcjambi 👋 

Are you behind a VPN or Proxy that is preventing the binary from downloading because I can correctly get the binary from my end?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by bblatnick32** [comment]
I get this error on Github Action CI all the time.  Seems like it times out intermittently.
